docs/konnectors-workflow.md

```diff
@@ -690,6 +704,23 @@ Location: https://domain.biapi.pro/2.0/auth/webview/reconnect
 At the end of flow, the user will be redirected to
 `https://jane-{slug}.cozy.example/?state={state}`.
 
+
+### BI Weview
+
+A new integration of Budget Insight is available for all the proposed banks.
+This is a workflow similar to OAuth called
+[webview](https://docs.budget-insight.com/reference/webview)
+
+It is possible to use this workflow for konnectors by registering an account
+type with the following parameter:
+
+- `grant_mode`, with `bi_webview+secret` as the value
+- `redirect_uri`, with an URL like `https://oauthcallback.mycozy.cloud/accounts/paypal/redirect`
+- `client_id`, with the client ID given by Budget Insight
+- `auth_endpoint`, with `https://{domain}.biapi.pro/2.0/auth/webview/fr/connect` (with the correct `domain`)
+- `reconnect_endpoint`, with `https://{domain}.biapi.pro/2.0/auth/webview/reconnect` (idem).
+
+
```

Comment:
It looks like a copy/paste error: the same block has been added twice. Can you fix it?